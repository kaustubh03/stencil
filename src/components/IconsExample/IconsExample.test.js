import React from 'react'
import renderer from 'react-test-renderer'
import Iconsexample from '.'

it('Iconsexample: default', () => {
  const component = renderer.create(<Iconsexample />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})