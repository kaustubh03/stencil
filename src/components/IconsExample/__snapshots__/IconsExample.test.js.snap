// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Iconsexample: default 1`] = `
<div
  className="wrapper"
>
  <div
    className="wrapper_container"
  >
    <div
      className="wrapper_heading"
    >
      React Icons Example
    </div>
    <h6>
      Also Have Support For Font Awesome, Material Design, Typicons, Github Octicons and Ionicons
    </h6>
    <br />
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12"
        >
          Lets go for a 
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 448 512"
            width="1em"
          >
            <path
              d="M368 96h-48V56c0-13.255-10.745-24-24-24H24C10.745 32 0 42.745 0 56v400c0 13.255 10.745 24 24 24h272c13.255 0 24-10.745 24-24v-42.11l80.606-35.977C429.396 365.063 448 336.388 448 304.86V176c0-44.112-35.888-80-80-80zm16 208.86a16.018 16.018 0 0 1-9.479 14.611L320 343.805V160h48c8.822 0 16 7.178 16 16v128.86zM208 384c-8.836 0-16-7.164-16-16V144c0-8.836 7.164-16 16-16s16 7.164 16 16v224c0 8.836-7.164 16-16 16zm-96 0c-8.836 0-16-7.164-16-16V144c0-8.836 7.164-16 16-16s16 7.164 16 16v224c0 8.836-7.164 16-16 16z"
            />
          </svg>
          ?
          <br />
        </div>
        <div
          className="col-md-12"
        >
          This is Another Icon 
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
          >
            <path
              d="M8 256c0 136.966 111.033 248 248 248s248-111.034 248-248S392.966 8 256 8 8 119.033 8 256zm248 184V72c101.705 0 184 82.311 184 184 0 101.705-82.311 184-184 184z"
            />
          </svg>
        </div>
        <div
          className="col-md-12"
        >
          This is Another Icon 
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
          >
            <path
              d="M32 448c0 17.7 14.3 32 32 32h384c17.7 0 32-14.3 32-32V160H32v288zm160-212c0-6.6 5.4-12 12-12h104c6.6 0 12 5.4 12 12v8c0 6.6-5.4 12-12 12H204c-6.6 0-12-5.4-12-12v-8zM480 32H32C14.3 32 0 46.3 0 64v48c0 8.8 7.2 16 16 16h480c8.8 0 16-7.2 16-16V64c0-17.7-14.3-32-32-32z"
            />
          </svg>
        </div>
        <div
          className="col-md-12"
        >
          This is Another Icon 
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 472 512"
            width="1em"
          >
            <path
              d="M422.5 202.9c30.7 0 33.5 53.1-.3 53.1h-10.8v44.3h-26.6v-97.4h37.7zM472 352.6C429.9 444.5 350.4 504 248 504 111 504 0 393 0 256S111 8 248 8c97.4 0 172.8 53.7 218.2 138.4l-186 108.8L472 352.6zm-38.5 12.5l-60.3-30.7c-27.1 44.3-70.4 71.4-122.4 71.4-82.5 0-149.2-66.7-149.2-148.9 0-82.5 66.7-149.2 149.2-149.2 48.4 0 88.9 23.5 116.9 63.4l59.5-34.6c-40.7-62.6-104.7-100-179.2-100-121.2 0-219.5 98.3-219.5 219.5S126.8 475.5 248 475.5c78.6 0 146.5-42.1 185.5-110.4z"
            />
          </svg>
        </div>
        <div
          className="col-md-12"
        >
          Checkout Official Docs
           
          <a
            href="http://react-icons.github.io/react-icons/fa.html"
          >
            here
          </a>
          .
        </div>
      </div>
    </div>
  </div>
</div>
`;
